frontend/src/diagram/sprotty/resize/siriusResize.ts

```diff
@@ -47,35 +47,37 @@ export class SiriusResizeCommand extends Command {
       let validSize = elementResize.newSize;
       let validPosition = elementResize.newPosition;
 
-      //If the element size is decreased, we compute a valid size and position to make sure that all children are still within the new bounds.
-      if (
-        element.children.filter((child) => this.isNode(child)).length > 0 &&
-        this.sizeDecreased(sizeDelta, positionDelta)
-      ) {
-        const [minTopLeft, maxBottomRight] = this.getChildrenLimits(element);
-        //A positive position delta means that the resize from the NW has been done toward the center of the figure (the size is decreased)
-        if (positionDelta.x > 0 || positionDelta.y > 0) {
-          const previousValidPositionDelta = validPositionDelta;
-          validPositionDelta = this.computeValidPositionDelta(validPositionDelta, minTopLeft);
-          validPosition = {
-            x: element.position.x + validPositionDelta.x,
-            y: element.position.y + validPositionDelta.y,
-          };
-          //If the valid position has been modified, we need to modify the size
-          const positionCorrection = {
-            x: previousValidPositionDelta.x - validPositionDelta.x,
-            y: previousValidPositionDelta.y - validPositionDelta.y,
-          };
-          validSize = {
-            width: validSize.width + positionCorrection.x,
-            height: validSize.height + positionCorrection.y,
-          };
-        }
-        if (sizeDelta.x < 0 || sizeDelta.y < 0) {
-          validSize = this.computeValidSize(validSize, maxBottomRight);
+      if (this.isNode(element)) {
```

Comment:
yes because the if part correspond to the logic to constraint node that ay contains others node and effectively the border node can not contain other nodes